Userland/Applications/PixelPaint/Tools/MoveTool.cpp

```diff
@@ -141,27 +141,24 @@ bool MoveTool::on_keydown(GUI::KeyEvent& event)
     if (m_scaling)
         return true;
 
-    if (event.modifiers() != 0)
-        return false;
```

Comment:
The old behavior was that using ctrl/alt caused the move to fail. Now, it moves the layer by 1 pixel. Maybe it's more consistent if only shift or no modifier works?

Reply:
I've pushed a change, which allows only the shift modifier to be pressed :^)